src/ovh-angular-sidebar-menu.provider.js

```diff
@@ -504,18 +504,36 @@ angular.module("ovh-angular-sidebar-menu").provider("SidebarMenu", function () {
         /* ----------  HELPERS  ----------*/
 
         function getItemStateInfos (item) {
-            var infos = {
+            if (_.isString(item.loadOnState)) {
+                return {
+                    included: $state.includes(item.loadOnState, item.loadOnStateParams),
+                    current: $state.is(item.loadOnState, item.loadOnStateParams)
+
+                };
+            }
+
+            if (_.isArray(item.loadOnState)) {
+                return {
+                    included: item.loadOnState.some(function (state) {
```

Comment:
```suggestion
                    included: item.loadOnState.some(state => $state.includes(state, item.loadOnStateParams))
```

Reply:
`babel` not available for this version of the module